modules/workers/instance.tf

```diff
@@ -0,0 +1,134 @@
+resource "oci_core_instance" "workers" {
+  for_each             = local.enabled_instances
+  availability_domain  = element(each.value.availability_domains, 1)
+  compartment_id       = each.value.compartment_id
+  display_name         = each.key
+  preserve_boot_volume = false
+  shape                = each.value.shape
+
+  defined_tags  = each.value.defined_tags
+  freeform_tags = each.value.freeform_tags
+
+  dynamic "shape_config" {
+    for_each = length(regexall("Flex", each.value.shape)) > 0 ? [1] : []
+    content {
+      ocpus = each.value.ocpus
+      memory_in_gbs = ( # If > 64GB memory/core, correct input to exactly 64GB memory/core
```

Comment:
i thought we did 32/core?

Reply:
Confirmed this is consistent with [main](https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/main/modules/oke/nodepools.tf#L78)